pycbc/waveform/compress.py

```diff
@@ -749,18 +749,28 @@ def write_to_hdf(self, fp, template_hash, root=None):
             Put the `compressed_waveforms` group in the given directory in the
             hdf file. If `None`, `compressed_waveforms` will be the root
             directory.
+        precision : {None, str}
+            Cast the saved parameters to the given precision before saving. If
+            None provided, will use whatever their current precision is. This
+            will raise an error if the parameters have single precision but the
+            requested precision is double.
```

Comment:
Same comment as below. Aren't the points getting generated as `double` by default, as we are doing everything in `double` fixing `dtype=complex128` always earlier in the code? So would the sentence ```This will raise an error if the parameters have single precision but the requested precision is double.``` be relevant?